project_service/test/flock-tests/index.html

```diff
@@ -1,9 +1,12 @@
 <html>
     <head>
         <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
     </head>
     <body>
         <div id="status"></div>
+
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```

Comment:
Does this mean production should have jquery, or should this be removed? Also why is it linked twice?

Reply:
Looks like JQuery is unused overall so both these script tags can probably be removed (confirmed with laura)